cypress/integration/ErrorHandling/TwoConsecutiveUserInput.spec.js

```diff
@@ -26,11 +26,11 @@ describe('ErrorHandling', () => {
     editDialogModal.ClickSaveCloseButton()
 
     modelPage.VerifyErrorIconForTrainDialogs()
-    train.VerifyErrorsFoundInTraining(`${String.fromCharCode(59412)}Hey`, 'world peace', "Sorry $name, I can't help you get $want")
+    train.VerifyErrorsFoundInTraining(`${common.errorIconCharacter}Hey`, 'world peace', "Sorry $name, I can't help you get $want")
 
     // - - - Open the same Train Dialog, validate and fix the errors. - - -
 
-    train.EditTraining(`${String.fromCharCode(59412)}Hey`, 'world peace', "Sorry $name, I can't help you get $want")
+    train.EditTraining(`${common.errorIconCharacter}Hey`, 'world peace', "Sorry $name, I can't help you get $want")
```

Comment:
Glad to see this gone.  Brings up related question from previous review.

It would be more appropriate to separate the assertion of first input from the validity. There is a dedicated selector `data-testid="train-dialogs-validity-indicator"` to that was added specifically for this.  It should find a dialog and assert it's validity. Not find a first input and assume it's invalid if the first input starts with some character.

From my understanding the icon can be present even when its not an error just other validity cases so I would prefer to keep naming of validityIcon.

Reply:
okay